### DISCUSSION TITLE
How to better set for deviated wells?

### DISCUSSION THREAD

**Discussion by HuihuiYangshell** [NatLabRockies/GEOPHIRES-X#238] [Q&A]
Like Fervo Red wells, the TVD is about 7700ft, Horizontal part is about 3250ft, and the MD is about 3336m. How could we better set the relative parameters, like gradient? segments? 
![Picture1](https://github.com/NREL/GEOPHIRES-X/assets/48022864/da66ac8a-4aca-4052-9c4d-f241b019484b)

**Comment by softwareengineerprogrammer** [comment]
@HuihuiYangshell you can set `Number of Segments`, `Gradient 1`, `Gradient 2`, etc., `Thickness 1`, `Thickness 2`, etc. up to 4 segments. Here is an example based on Fervo Project Red with the approximate gradients from your image: https://gtp.scientificwebservices.com/geophires/?shared-geophires-result-id=bbb10f31-0a0f-47c7-87df-94329ad55886

Screenshot for reference:

<img width="1345" alt="Screenshot 2024-06-19 at 06 20 05" src="https://github.com/NREL/GEOPHIRES-X/assets/4056124/b1919ef7-df5c-4986-831e-9083a3d5295c">
